src/components/sidebar/InputContainer.tsx

```diff
@@ -7,10 +7,18 @@ interface IInputProps {
     placeholder?: string;
     className?: string;
     disabled?: boolean;
+    onChange?: Function;
+    value?: string;
 }
 
 class InputContainer extends React.Component<IInputProps> {
     public render(): any {
+        const onChange = (e: React.FormEvent<HTMLInputElement>) => {
```

Comment:
I'm not sure if this is the correct way of implementing input field binding. Smaller cards would be preferred.

Reply:
This is how React documentation does it, I'm open to better alternatives.
https://reactjs.org/docs/forms.html